### ISSUE TITLE
Fine-tune about reference

### ISSUE THREAD

**Issue by guangjh** [Z1zs/MMUnlearner#5]
I'm reproduce reference model (only fine-tune on retain set),but it shows high accuracy on the forget set and low accuracy on the retain set. I don't understand why this is happening.I did't do any changes about your code.  Can you help me explain?
--forget_split_ratio=5 \
    --batch_size=4  \
    --lr=2e-5  \
    --num_epochs=4 \
{
    "Forget Set Results": {
        "fill_in_the_blank": {
            "image_textual_accuracy": 10.416666666666668,
            "pure_text_accuracy": 2.083333333333333
        },
        "classification": {
            "Image-Textual Question Accuracy": 40.0,
            "Pure Text Question Accuracy": 5.0
        },
        "generation": {
            "Average ROUGE-1 (Image_Textual)": 0.33854566022213073,
            "Average ROUGE-2 (Image_Textual)": 0.19829803529803527,
            "Average ROUGE-L (Image_Textual)": 0.3342879571408982,
            "Average BLEU (Image_Textual)": 0.10984906782294854,
            "Average ROUGE-1 (Pure_Text)": 0.4907138370745182,
            "Average ROUGE-2 (Pure_Text)": 0.3128159609568278,
            "Average ROUGE-L (Pure_Text)": 0.47660857391662353,
            "Average BLEU (Pure_Text)": 0.1956597354037594
        }
    }
}
Retain Set (shared dataset) Results:
{'image_textual_accuracy': 6.434599156118144, 'pure_text_accuracy': 5.485232067510549}
{'Image-Textual Question Accuracy': 29.429175475687103, 'Pure Text Question Accuracy': 2.367864693446089}
{'Average ROUGE-1 (Image_Textual)': 0.28032693616252985, 'Average ROUGE-2 (Image_Textual)': 0.166087290352457, 'Average ROUGE-L (Image_Textual)': 0.27766338963115517, 'Average BLEU (Image_Textual)': 0.08948267810169519, 'Average ROUGE-1 (Pure_Text)': 0.5570855221950662, 'Average ROUGE-2 (Pure_Text)': 0.41675336043655725, 'Average ROUGE-L (Pure_Text)': 0.5418433658599914, 'Average BLEU (Pure_Text)': 0.28541863805779444}
{'Retain Set (shared dataset) Results': {'fill_in_the_blank': {'image_textual_accuracy': 6.434599156118144, 'pure_text_accuracy': 5.485232067510549}, 'classification': {'Image-Textual Question Accuracy': 29.429175475687103, 'Pure Text Question Accuracy': 2.367864693446089}, 'generation': {'Average ROUGE-1 (Image_Textual)': 0.28032693616252985, 'Average ROUGE-2 (Image_Textual)': 0.166087290352457, 'Average ROUGE-L (Image_Textual)': 0.27766338963115517, 'Average BLEU (Image_Textual)': 0.08948267810169519, 'Average ROUGE-1 (Pure_Text)': 0.5570855221950662, 'Average ROUGE-2 (Pure_Text)': 0.41675336043655725, 'Average ROUGE-L (Pure_Text)': 0.5418433658599914, 'Average BLEU (Pure_Text)': 0.28541863805779444}}}

**Comment by Z1zs** [comment]
Hi @guangjh 
The fine-tuning dynamics of MLLMs is relatively complex and such phenomena does have a chance to happen(https://arxiv.org/abs/2402.16832). 
We recheck our code to make sure that its logic's correct and aligns with MLLMU-Bench repo(https://github.com/franciscoliu/MLLMU-Bench). My personal suggestion is to adjust the epoch num and learning_rate to see if the results remain. You may also refer to MLLMU-Bench repo to find more information.

**Comment by guangjh** [comment]
Hello, I’ve tried many times but still haven’t been able to properly train the reference model using only the retain set. The reference model I trained always achieves very high accuracy on the forget set, and I’m not sure why. Could you please open-sourcing the reference model for the MLLMU dataset? I look forward to your reply.

**Comment by Z1zs** [comment]
Hi @guangjh 
Sorry but we can't release the model weights since they are all stored in company server and it has been a long time since publication. However you can refer to MLLMU-Bench repo for help about their baselines if applicable.
